test/e2e/distributedtracing/distributedtracing_test.go

```diff
@@ -88,7 +92,7 @@ func TestMain(m *testing.M) {
 		resp.body.count == 1
 	}`
 	module := fmt.Sprintf(moduleFmt, ts.URL)
-	pluginsManager, err := e2e.TestAuthzServerWithWithOpts(module, "envoy/authz/allow", ":9191", plugins.WithTracerProvider(tracerProvider))
+	pluginsManager, err := e2e.TestAuthzServerWithWithOpts(module, "envoy/authz/allow", ":9191", plugins.WithTracerProvider(tracerProvider), plugins.ConsoleLogger(consoleLogger))
```

Comment:
Do you intend to use the console logger somewhere in this test?

Reply:
@ashutosh-narkar  , yes. 

I'm using it to capture the log entries, identify a decision log in them, and then assert for the presence of trace_id and span_id in the decision log. 

see line 155 to 159